lib/cylc/batch_sys_manager.py

```diff
@@ -72,9 +72,13 @@
       "batch_sys.SUBMIT_CMD".
 
 batch_sys.CAN_KILL_PROC_GROUP
-    * A boolean to indicate whether it is possible to kill the job by sending
+    * A boolean to indicate whether it is possible to kill a job by sending
       a signal to its Unix process group.
 
+batch_sys.CAN_POLL_PROC_GROUP
+    * A boolean to indicate whether it is necessary to poll a job by its PID
+      as well as the job ID.
+
```

Comment:
Should this be `SHOULD_POLL_PROCESS_GROUP`?  (you "can" poll any job by PID, but for at we need to do it)